mainapp/models.py

```diff
@@ -693,3 +745,34 @@ class Hospital(models.Model):
 
     def __str__(self):
         return self.name + ' - ' + self.designation
+
+    @classmethod
+    def count(cls):
+        return cls._default_manager.count()
+
+
+@receiver(post_save, sender=Request)
+@receiver(post_save, sender=RescueCamp)
+@receiver(post_save, sender=Announcements)
+@receiver(post_save, sender=Contributor)
+@receiver(post_save, sender=DistrictNeed)
+@receiver(post_save, sender=Volunteer)
+@receiver(post_save, sender=DistrictManager)
+@receiver(post_save, sender=Hospital)
+@receiver(post_save, sender=PrivateRescueCamp)
+@receiver(post_save, sender=CollectionCenter)
+def expire_home_page_count_cache(sender, created, **kwargs):
```

Comment:
Not sure if we should have a single cache key for all counts. The update frequency across all the above models would be much higher than for individual models.

Reply:
That is an aggressive mode which is not enabled by default.